### PR TITLE
[WIP][DO NOT MERGE] Remove dependency model

### PR DESCRIPTION
**Pull request by livarcocc**
Removing dependency model from the CLI.

cc @brthor @eerhardt @piotrpMSFT

### CODE REVIEW

File: src/Microsoft.DotNet.ProjectModel/RuntimeEnvironmentRidExtensions.cs [hunk 1]
```diff
@@ -1,5 +1,7 @@
-﻿using System;
+﻿
+using System;
```
Comment (jasonwilliams200OK) [inline src/Microsoft.DotNet.ProjectModel/RuntimeEnvironmentRidExtensions.cs]: nit: this file is missing copyright header.

**Comment by jasonwilliams200OK** [comment]
Is the new dependency model package closed source?

**Comment by eerhardt** [comment]
No it isn't closed source. It just moved to another repo:  https://github.com/dotnet/core-setup/tree/master/src/Microsoft.Extensions.DependencyModel

**Comment by livarcocc** [comment]
@dotnet-bot Test this please.

**Comment by livarcocc** [comment]
CLosing this one. I will move this PR to feature/msbuild branch.

**Pull request closed without merging**